[2024-04-02T16:42:47.940+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_cron.third_task manual__2024-04-02T16:42:45.108863+00:00 [queued]>
[2024-04-02T16:42:47.949+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_cron.third_task manual__2024-04-02T16:42:45.108863+00:00 [queued]>
[2024-04-02T16:42:47.950+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 4
[2024-04-02T16:42:47.963+0000] {taskinstance.py:2217} INFO - Executing <Task(BashOperator): third_task> on 2024-04-02 16:42:45.108863+00:00
[2024-04-02T16:42:47.968+0000] {standard_task_runner.py:60} INFO - Started process 885 to run task
[2024-04-02T16:42:47.970+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'first_cron', 'third_task', 'manual__2024-04-02T16:42:45.108863+00:00', '--job-id', '193', '--raw', '--subdir', 'DAGS_FOLDER/first_crnschedular.py', '--cfg-path', '/tmp/tmpr4c_c8n2']
[2024-04-02T16:42:47.971+0000] {standard_task_runner.py:88} INFO - Job 193: Subtask third_task
[2024-04-02T16:42:48.009+0000] {task_command.py:423} INFO - Running <TaskInstance: first_cron.third_task manual__2024-04-02T16:42:45.108863+00:00 [running]> on host 5c9eefa8eb82
[2024-04-02T16:42:48.076+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='first_cron' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2024-04-02T16:42:45.108863+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-02T16:42:45.108863+00:00'
[2024-04-02T16:42:48.077+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-04-02T16:42:48.078+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is my third task']
[2024-04-02T16:42:48.087+0000] {subprocess.py:86} INFO - Output:
[2024-04-02T16:42:48.088+0000] {subprocess.py:93} INFO - hello world, this is my third task
[2024-04-02T16:42:48.089+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-04-02T16:42:48.108+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=first_cron, task_id=third_task, execution_date=20240402T164245, start_date=20240402T164247, end_date=20240402T164248
[2024-04-02T16:42:48.143+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-02T16:42:48.157+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
